node/node_test.go

```diff
@@ -1360,3 +1362,99 @@ func TestNode_GetAccountAccountExistsShouldReturn(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, accnt, recovAccnt)
 }
+
+func TestNode_AppStatusHandlerWithPrometheusIncrementAndDecrementMetric(t *testing.T) {
```

Comment:
Why do you use prometheus impl to test this?

Reply:
it doesn't use anymore